docs/common-coreboot-docs/dasharo_tools_suite.md

```diff
@@ -162,6 +181,150 @@ fwupdmgr refresh
 fwupdmgr update
 ```
 
+## Dasharo EC Transition
+
+DTS allows to perform full EC transition from vendor to Dasharo/EC FW. For now
```

Comment:
Maybe we should use the full terms, i. e. Embedded Controller, firmware? At least the first time we mention something?

Reply:
fixed https://github.com/Dasharo/docs/pull/291/commits/1d137963d289dfceb153d241d870f5174335a96b#diff-b5545f47decb8260688b188363790fc5aef4ea727b9c363329f070d1fa3664a4R186